tests/Oryx.Integration.Tests/Database/PulledBuildImageTypeTest.cs

```diff
@@ -0,0 +1,69 @@
+﻿// --------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+// --------------------------------------------------------------------------------------------
+
+using System;
+using Microsoft.Oryx.Tests.Common;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Oryx.Integration.Tests
+{
+    [Trait("category", "db")]
```

Comment:
@kichalla .. i added this as a db category .. it doesn't matter in general .. because all the categories will be pulling and tagging in the same way .. so if db category test fails it ultimately reflects pulling and tagging is wrong for other categories as well.